.gitignore

```diff
@@ -9,3 +9,4 @@
 /website/src/draft-js/lib/
 /website/src/draft-js/docs/
 npm-debug.log
+.idea/
```

Comment:
Oops, sorry, that's an artifact of IntelliJ editors, I added it so config files wouldn't be thrown in with the changes. I'll remove it...